### ISSUE TITLE
BloomFilters does not work on Julia v1.0

### ISSUE THREAD

**Issue by aj-monk** [johnmyleswhite/BloomFilters.jl#22]
```julia
v1.0) pkg> activate .

(Test) pkg> add https://github.com/johnmyleswhite/BloomFilters.jl
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
  Updating git-repo `https://github.com/johnmyleswhite/BloomFilters.jl`
 Resolving package versions...
  Updating `Project.toml`
  [d3d7f49f] + BloomFilters v0.2.0+ #master (https://github.com/johnmyleswhite/BloomFilters.jl)
  Updating `Manifest.toml`
  [d3d7f49f] + BloomFilters v0.2.0+ #master (https://github.com/johnmyleswhite/BloomFilters.jl)
  [34da2185] + Compat v1.1.0
  [2a0f44e3] + Base64 
  [ade2ca70] + Dates 
  [8bb1440f] + DelimitedFiles 
  [8ba89e20] + Distributed 
  [b77e0a4c] + InteractiveUtils 
  [76f85450] + LibGit2 
  [8f399da3] + Libdl 
  [37e2e46d] + LinearAlgebra 
  [56ddb016] + Logging 
  [d6f4376e] + Markdown 
  [a63ad114] + Mmap 
  [44cfe95a] + Pkg 
  [de0858da] + Printf 
  [3fa0cd96] + REPL 
  [9a3f8284] + Random 
  [ea8e919c] + SHA 
  [9e88b42a] + Serialization 
  [1a1011a3] + SharedArrays 
  [6462fe0b] + Sockets 
  [2f01184e] + SparseArrays 
  [10745b16] + Statistics 
  [8dfed614] + Test 
  [cf7118a7] + UUIDs 
  [4ec0a83e] + Unicode 

julia> using BloomFilters
[ Info: Precompiling BloomFilters [d3d7f49f-8593-5289-ba71-9c4f4848f489]
ERROR: LoadError: LoadError: UndefVarError: contains not defined
Stacktrace:
 [1] getproperty(::Module, ::Symbol) at ./sysimg.jl:13
 [2] top-level scope at none:0
 [3] include at ./boot.jl:317 [inlined]
 [4] include_relative(::Module, ::String) at ./loading.jl:1038
 [5] include at ./sysimg.jl:29 [inlined]
 [6] include(::String) at /home/ajaym/.julia/packages/BloomFilters/6zNxm/src/BloomFilters.jl:1
 [7] top-level scope at none:0
 [8] include at ./boot.jl:317 [inlined]
 [9] include_relative(::Module, ::String) at ./loading.jl:1038
 [10] include(::Module, ::String) at ./sysimg.jl:29
 [11] top-level scope at none:2
 [12] eval at ./boot.jl:319 [inlined]
 [13] eval(::Expr) at ./client.jl:389
 [14] top-level scope at ./none:3
in expression starting at /home/ajaym/.julia/packages/BloomFilters/6zNxm/src/bloom-filter.jl:122
in expression starting at /home/ajaym/.julia/packages/BloomFilters/6zNxm/src/BloomFilters.jl:3
ERROR: Failed to precompile BloomFilters [d3d7f49f-8593-5289-ba71-9c4f4848f489] to /home/ajaym/.julia/compiled/v1.0/BloomFilters/ddTJ4.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] macro expansion at ./logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at ./loading.jl:1184
 [4] _require(::Base.PkgId) at ./logging.jl:311
 [5] require(::Base.PkgId) at ./loading.jl:852
 [6] macro expansion at ./logging.jl:311 [inlined]
 [7] require(::Module, ::Symbol) at ./loading.jl:834
```

**Comment by aj-monk** [comment]
_contains_ has been removed in v1.0. Using _in_ instead of _contains_ works.

diff --git a/src/bloom-filter.jl b/src/bloom-filter.jl
index 661e021..0520bd2 100644
--- a/src/bloom-filter.jl
+++ b/src/bloom-filter.jl
@@ -119,7 +119,7 @@ function add!(bf::BloomFilter, key::Any)
     end
 end
 
```diff
-function Base.contains(bf::BloomFilter, key::Any)
+function Base.in(bf::BloomFilter, key::Any)
     hashes = hash_n(key, bf.k, bf.n_bits)
     for h in hashes
         if bf.array[h] != 1
@@ -138,11 +138,11 @@ function add!(bf::BloomFilter, keys::Vector{Any})
     end
 end
 
-function Base.contains(bf::BloomFilter, keys::Vector{Any})
+function Base.in(bf::BloomFilter, keys::Vector{Any})
     m = length(keys)
     res = falses(m)
     for i in 1:m
-        res[i] = contains(bf, keys[i])
+        res[i] = in(bf, keys[i])
     end
     return res
 end
```

**Comment by aj-monk** [comment]
```diff
diff --git a/src/BloomFilters.jl b/src/BloomFilters.jl
index 75115a7..fca6f57 100644
--- a/src/BloomFilters.jl
+++ b/src/BloomFilters.jl
@@ -1,4 +1,4 @@
 module BloomFilters
-       export BloomFilter, add!
+       export BloomFilter, add!, contains
        include("bloom-filter.jl")
 end
diff --git a/src/bloom-filter.jl b/src/bloom-filter.jl
index 661e021..8d6d2a4 100644
--- a/src/bloom-filter.jl
+++ b/src/bloom-filter.jl
@@ -97,7 +97,7 @@ end
 function BloomFilter(mmap_handle::IOStream, capacity::Int, error_rate::Float64)
     bits_per_elem = round(Int, ceil(-1.0 * (log(error_rate) / (log(2) ^ 2))))
     k_hashes = round(Int, log(2) * bits_per_elem)  # Note: ceil() would be strictly more conservative
-    n_bits = capacity * bits_per_elem    
+    n_bits = capacity * bits_per_elem
     mb = mmap(mmap_handle, BitArray, n_bits)
     BloomFilter(mb, k_hashes, capacity, error_rate, n_bits, mmap_handle.name)
 end
@@ -111,7 +111,7 @@ function BloomFilter(mmap_string::AbstractString, capacity::Int, error_rate::Flo
     BloomFilter(mmap_handle, capacity, error_rate)
 end
 
-### Bloom filter functions: insert!, add! (alias to insert), contains, and show
+### Bloom filter functions: insert!, add! (alias to insert), in, contains, and show
 function add!(bf::BloomFilter, key::Any)
     hashes = hash_n(key, bf.k, bf.n_bits)
     for h in hashes
@@ -119,7 +119,7 @@ function add!(bf::BloomFilter, key::Any)
     end
 end
 
-function Base.contains(bf::BloomFilter, key::Any)
+function Base.in(key::Any, bf::BloomFilter)
     hashes = hash_n(key, bf.k, bf.n_bits)
     for h in hashes
         if bf.array[h] != 1
@@ -129,7 +129,7 @@ function Base.contains(bf::BloomFilter, key::Any)
     return true
 end
 
-Base.in(key::Any, bf::BloomFilter) = contains(bf, key)
+contains(bf::BloomFilter, key::Any) = in(key, bf)
 
 # Vector variants
 function add!(bf::BloomFilter, keys::Vector{Any})
@@ -138,7 +138,7 @@ function add!(bf::BloomFilter, keys::Vector{Any})
     end
 end
 
-function Base.contains(bf::BloomFilter, keys::Vector{Any})
+function contains(bf::BloomFilter, keys::Vector{Any})
     m = length(keys)
     res = falses(m)
     for i in 1:m
```

**Comment by boydgreenfield** [comment]
Fixed in #24.
